database/raw.sql

```diff
@@ -0,0 +1,1590 @@
+    , ev.iab__reason
+    , ev.iab__spider_or_robot
+    , ev.yauaa__device_class
+    , ev.yauaa__agent_class
+    , ev.yauaa__agent_name
+    , ev.yauaa__agent_name_version
+    , ev.yauaa__agent_name_version_major
+    , ev.yauaa__agent_version
+    , ev.yauaa__agent_version_major
+    , ev.yauaa__device_brand
+    , ev.yauaa__device_name
+    , ev.yauaa__device_version
+    , ev.yauaa__layout_engine_class
+    , ev.yauaa__layout_engine_name
+    , ev.yauaa__layout_engine_name_version
+    , ev.yauaa__layout_engine_name_version_major
+    , ev.yauaa__layout_engine_version
+    , ev.yauaa__layout_engine_version_major
+    , ev.yauaa__operating_system_class
+    , ev.yauaa__operating_system_name
+    , ev.yauaa__operating_system_name_version
+    , ev.yauaa__operating_system_version
+      , ev.ua__useragent_family
+      , ev.ua__useragent_major
+      , ev.ua__useragent_minor
+      , ev.ua__useragent_patch
+      , ev.ua__useragent_version
+      , ev.ua__os_family
+      , ev.ua__os_major
+      , ev.ua__os_minor
+      , ev.ua__os_patch
+      , ev.ua__os_patch_minor
+      , ev.ua__os_version
+      , ev.ua__device_family
+  , ev.app__build
+  , ev.app__version
+    , ev.geo__latitude
+    , ev.geo__longitude
+    , ev.geo__latitude_longitude_accuracy
+    , ev.geo__altitude
+    , ev.geo__altitude_accuracy
+    , ev.geo__bearing
+    , ev.geo__speed
+      , ev.screen__id
+      , ev.screen__name
+      , ev.screen__activity
+      , ev.screen__fragment
+      , ev.screen__top_view_controller
+      , ev.screen__type
+      , ev.screen__view_controller
+    , ev.mobile__device_manufacturer
+    , ev.mobile__device_model
+    , ev.mobile__os_type
+    , ev.mobile__os_version
+    , ev.mobile__android_idfa
+    , ev.mobile__apple_idfa
+    , ev.mobile__apple_idfv
+    , ev.mobile__carrier
+    , ev.mobile__open_idfa
+    , ev.mobile__network_technology
+    , ev.mobile__network_type
+    , ev.mobile__physical_memory
+    , ev.mobile__system_available_memory
+    , ev.mobile__app_available_memory
+    , ev.mobile__battery_level
+    , ev.mobile__battery_state
+    , ev.mobile__low_power_mode
+    , ev.mobile__available_storage
+    , ev.mobile__total_storage
+    , ev.mobile__is_portrait
+    , ev.mobile__resolution
+    , ev.mobile__scale
+    , ev.mobile__language
+    , ev.mobile__app_set_id
+    , ev.mobile__app_set_id_scope
+      ,ev.v_collector
+      ,event_id as event_id2
+
+    from unified_events as ev
+
+    -- removes ga4 source categories
+    where ev.event_name in ('page_view', 'screen_view')
+    and ev.view_id is not null
+    and coalesce(iab__spider_or_robot, False ) = False
+  and ev.useragent not similar to '.*(bot|crawl|slurp|spider|archiv|spinn|sniff|seo|audit|survey|pingdom|worm|capture|(browser|screen)shots|analyz|index|thumb|check|facebook|PingdomBot|PhantomJS|YandexBot|Twitterbot|a_archiver|facebookexternalhit|Bingbot|BingPreview|Googlebot|Baiduspider|360(Spider|User-agent)|semalt).*'
```

Comment:
Wasn't this already done in `base` when building `unified_events`, so redundant here?
If not redundant for some reason, should it consider the `GETVARIABLE('ua_bot_filter')` like we do there?

Ditto below in several places.